brick/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	var a, b float64
+	fmt.Println("Enter the hole dimensions (2 numbers):")
+	fmt.Scan(&a, &b)
+	var p, q, r float64
+	fmt.Println("Enter the brick dimensions (3 numbers):")
+	fmt.Scan(&p, &q, &r)
+	if p < r && p > q && r > q {
+		p, r, q = q, p, r
+	} else if r < p && q < p && r < q {
+		p, r, q = r, q, p
+	} else if p < q && q < r && p < r {
+		r, q = q, r
+	} else if q < r && r < p && q < p {
+		p, q = q, p
+	} else if r < p && p < q && r < q {
+		p, r = r, p
+	}
+	if p <= a && r <= b || p <= b && r <= a {
```

Comment:
nit: You don't need all these comparisons. It's enough to compare two smallest dimension of the brick to the hole.